tests/diviner/test_diviner_model_export.py

```diff
@@ -0,0 +1,479 @@
+import pathlib
+import os
+import pytest
+from unittest import mock
+
+import yaml
+import numpy as np
+import pandas as pd
+
+from diviner import GroupedProphet, GroupedPmdarima
+from diviner.utils.example_utils import example_data_generator
+
+from mlflow.exceptions import MlflowException
+from mlflow.models import infer_signature, Model
+from mlflow.models.utils import _read_example
+import mlflow.pyfunc.scoring_server as pyfunc_scoring_server
+from mlflow.store.artifact.s3_artifact_repo import S3ArtifactRepository
+from mlflow.tracking.artifact_utils import _download_artifact_from_uri
+from mlflow.tracking._model_registry import DEFAULT_AWAIT_MAX_SLEEP_SECONDS
+from mlflow.utils.environment import _mlflow_conda_env
+from mlflow.utils.file_utils import TempDir
+from mlflow.utils.model_utils import _get_flavor_configuration
+
+import mlflow.diviner
+from mlflow import pyfunc
+
+from tests.helper_functions import mock_s3_bucket  # pylint: disable=unused-import
+from tests.helper_functions import (
+    _compare_conda_env_requirements,
+    _assert_pip_requirements,
+    pyfunc_serve_and_score_model,
+    _is_available_on_pypi,
+    _compare_logged_code_paths,
+)
+
+
+DS_FORMAT = "%Y-%m-%dT%H:%M:%S"
+EXTRA_PYFUNC_SERVING_TEST_ARGS = [] if _is_available_on_pypi("diviner") else ["--no-conda"]
+
+
+@pytest.fixture
+def model_path(tmpdir):
+    return str(pathlib.Path(tmpdir).joinpath("model"))
+
+
+@pytest.fixture(scope="session")
+def diviner_data():
+    return example_data_generator.generate_example_data(
+        column_count=3, series_count=4, series_size=365 * 4, start_dt="2019-01-01", days_period=1
+    )
+
+
+@pytest.fixture(scope="session")
```

Comment:
```suggestion
@pytest.fixture(scope="module")
```

Same here (https://github.com/mlflow/mlflow/pull/5553/files#r838061013)

Reply:
Changed all 3 instances to 'module' scoping